tfe/tool_helpers.go

```diff
@@ -0,0 +1,42 @@
+package tfe
+
+import (
+	"fmt"
+
+	tfe "github.com/hashicorp/go-tfe"
+)
+
+// fetchTerraformVersionID returns a Terraform Version ID for the given Terraform version number
+func fetchTerraformVersionID(version string, client *tfe.Client) (string, error) {
+	versionID := ""
+	pageNum := 0
+
+found:
+	for {
+		versions, err := client.Admin.TerraformVersions.List(ctx, tfe.AdminTerraformVersionsListOptions{
```

Comment:
since this requires admin access, it might not be a general purpose helper. I'd suggest just keeping this function within tfe/resource_tfe_terraform_version.go

Reply:
My hesitation with this is that it deviates from the rest of the code organization... even though moving the function to another file changes nothing (since it's the same module). Alternatively I can rename `tool_helpers.go` to `admin_helpers.go` which better indicates these are helpers for admins ? WDYT?

Since I think this is the first "Admin-Only" resource we've added to the provider,  it's open ended on how best to structure this.